src/equations/inviscid_burgers_1d.jl

```diff
@@ -116,5 +172,10 @@ end
 @inline energy_total(u::Real, ::InviscidBurgersEquation1D) = 0.5 * u^2
 @inline energy_total(u, equation::InviscidBurgersEquation1D) = energy_total(u[1], equation)
 
+# For shock capturing
```

Comment:
Isn't this `first`?

Reply:
Sorry, I don't get what you are referring to.

Reply:
[`Base.first`](https://docs.julialang.org/en/v1/base/collections/#Base.first)

Reply:
As in this example
https://github.com/trixi-framework/Trixi.jl/blob/00e2e06f8e3aeeb088a2753c2ae2360bd8d9e53b/examples/tree_2d_dgsem/elixir_kpp.jl#L81

Reply:
Ah okay - I can remove that routine if you would like to keep the equation file short.
IMHO, however, there is some value of the "expressiveness" of the explicit indicator function gained since it provides a simplification of the more advanced indicators of e.g. compressible euler equations.

Reply:
It's defined here:
https://github.com/trixi-framework/Trixi.jl/blob/3ea50d1d108d9c3b4d5ce463a40a20bfa7937633/src/equations/equations.jl#L229
I do not like implementing the same function multiple times - but there may be some advantage in terms of repeating code as examples. What do you think, @sloede?

Reply:
Ah okay, that changes things - I thought that the `first`here

https://github.com/trixi-framework/Trixi.jl/blob/00e2e06f8e3aeeb088a2753c2ae2360bd8d9e53b/examples/tree_2d_dgsem/elixir_kpp.jl#L81

is exactly the `first` from the Julia [`Base`](https://docs.julialang.org/en/v1/base/collections/#Base.first). 
Having an "own" implementation of `first` in Trixi makes it somewhat more clear.